src/raps/unlockAndSwap.ts

```diff
@@ -1,9 +1,9 @@
 import {
   ALLOWS_PERMIT,
   ETH_ADDRESS as ETH_ADDRESS_AGGREGATOR,
-  PermitSupportedTokenList,
-  WRAPPED_ASSET,
   getRainbowRouterContractAddress,
+  PermitSupportedTokenList,
+  SwapType,
```

Comment:
imports that can be removed:
- `SwapType` on line 6
- `ETH_ADDRESS` on line 14